sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryHelpers.java

```diff
@@ -411,6 +411,21 @@ public static TableReference parseTableSpec(String tableSpec) {
     return ref.setDatasetId(match.group("DATASET")).setTableId(match.group("TABLE"));
   }
 
+  public static TableReference parseTableUrn(String tableUrn) {
+    Matcher match = BigQueryIO.TABLE_URN_SPEC.matcher(tableUrn);
+    if (!match.matches()) {
+      throw new IllegalArgumentException(
+          "Table reference is not in projects/[project_id]/datasets/[dataset_id]/tables/[table_id] "
```

Comment:
Why would this be in this format (and not "project:dataset.table") ?

Probably good to link to some documentation that points to the convention used here.

Reply:
+1. My impression is that the BQ table naming convention is `project:dataset_name.table_name`? Though I did a quick search and didn't find the exact BQ naming convention.

Reply:
Vortex expects this format, and doesn't accept project:dataset.table